src/Port.js

```diff
@@ -0,0 +1,13 @@
+function Port(portName){
+    this.portName = portName;
+    this.ships = [];
+    this.addShip = function(ship){
+        this.ships.push(ship);
+    };
+    this.removeShip = function(ship){
+        const indexOfShip = this.ships.indexOf(ship);
+        this.ships.splice(indexOfShip, 1);
```

Comment:
Could the variable created here be removed like so?
```suggestion
        this.ships.splice(this.ships.indexOf(ship), 1);
```